docs/guidance/model/milestones.md

```diff
@@ -0,0 +1,141 @@
+
+```eval_rst
+
+.. jsoninclude:: ../../examples/milestones/planning-tender-milestones.json
+   :jsonpointer: 
+   :expand: releases, planning, milestones, tender, milestones
+   :title: planning
+
+.. jsoninclude:: ../../examples/milestones/planning-tender-milestones-2.json
+   :jsonpointer: 
+   :expand: releases, planning, milestones, tender, milestones
+   :title: tender
+
+```
+
+### Contract implementation milestones
+
+#### 1. Project data
+
+This worked example shows how milestones for project commencement and project completion should be updated in OCDS.
+
+The example includes three updates to a single contracting process (ocds-213czf-000-00001):
+
+* A contract is awarded to a user committee with milestones for project commencement and project completion.
+* The project starts later than expected. The project commencement milestone is updated with the actual start date and the project completion milestone `status` is set to 'notMet'.
+* The project is completed earlier than expected. The project completion milestone is updated with the actual completion date.
+
+The OCDS releases for each update are shown below. 
+
+```eval_rst
+
+.. jsoninclude:: ../../examples/milestones/implementation-milestones-1.json
+   :jsonpointer: 
+   :expand: releases, contracts, implementation, milestones
+   :title: implementation
+
+```
+
+```eval_rst
+
+.. jsoninclude:: ../../examples/milestones/implementation-milestones-2.json
+   :jsonpointer: 
+   :expand: releases, contracts, implementation, milestones
+   :title: implementation-update-1
+
+```
+
+```eval_rst
+
+.. jsoninclude:: ../../examples/milestones/implementation-milestones-3.json
+   :jsonpointer: 
+   :expand: releases, contracts, implementation, milestones
+   :title: implementation-update-2
+
+```
+
+#### 2. Delivery and financing data
+
+This example shows how milestones can be used to keep track of delivery and financing (payment) data in a contracting process.
+
+The procuring entity in the example, the Ministry of Border and Tribal Affairs,
+is managing the procurement for the mantainance of a high school building.
+Milestones have been set for the delivery and payment of goods and services
+related to the project. In the first release shown below, contract information
+is released along its implementation milestones.
+
+Once the first milestone has been reached ("Finish the exterior and interior walls"), a new release is created with  the related milestone `status` set to 'met' and the relevant dates added to the `dateMet` and `dateModified` fields.
+
+In a second update, the construction company has received payment for the work done so far, and the milestone for the wall restoration with type 'financing' is updated in a new release. 
+A new `transaction` is also disclosed, with the amount paid to the company.
```

Comment:
Please update in-line with the structure I suggested for the first two examples. Thanks!